Makefile

```diff
@@ -2,13 +2,33 @@
 
 REPO   := docker.stackable.tech/stackable
 TAG    := $(shell git rev-parse --short HEAD)
+ARCH   := $(shell arch)
 
 define push
 	docker push --all-tags ${REPO}/$(1)
 endef
 
 define build
-	@docker build --force-rm -t "${REPO}/${1}:${TAG}" -t "${REPO}/${1}:latest" -f $(1)/Dockerfile .
+	@docker build --force-rm -t "${REPO}/${1}:${TAG}-${ARCH}" -t "${REPO}/${1}:latest-${ARCH}" -f $(1)/Dockerfile .
+endef
+
+# Tag for arm64 is fixed since this will run on gh ubuntu-latest
+define manifest
+	@docker manifest create "${REPO}/${1}:latest" ${REPO}/${1}:latest-${ARCH} ${REPO}/${1}:latest-aarch64
```

Comment:
Won't this break on ARM since it only links in aarch64 twice?

Reply:
Yeah it would, but I assume this only is meant to be for the build-pipeline. The create manifest list is meant to only run on ubuntu-latest, therefore I went down the easy path.

Not sufficient?

Reply:
I still don't see a reason to not just hard-code amd64 (like you already do for arm64).

Reply:
Ah, yeah we could do that. No pains with that

Reply:
Hardcoded arch wherever possible